Installing from source on mac fails

When trying to install iptux on OS X during this step
`mkdir build && cd build && cmake .. && make`
i get the following error:
CMake Error: The source directory does not appear to contain CMakeLists.txt.

--- a/README.md
+++ b/README.md
@@ -94,8 +94,8 @@ brew install gst-plugins-good
 # endif
 git clone git://github.com/iptux-src/iptux.git
 cd iptux
-mkdir build && cd build && cmake .. && make
-sudo make install
+meson builddir
+ninja -C builddir install
 iptux
 ```
 

